src/color.rs

```diff
@@ -61,11 +61,6 @@ impl ColorU8 {
         self.alpha() == ALPHA_U8_OPAQUE
     }
 
-    /// Returns the value as a primitive type.
-    pub const fn get(self) -> u32 {
```

Comment:
Should we keep it, but return `[u8; 4]` instead?

Reply:
I removed the `get` functions because I see no reason to make the internal representation part of the public api. The only guarantee we need to make is that when you cast it to bytes, the layout is RGBA. Whether that comes from `u32` or `[u8; 4]` is irrelevant. But maybe it is useful to keep for convenience? I'm not sure.